apps/src/utils.js

```diff
@@ -743,6 +743,15 @@ export function stringifyQueryParams(params) {
   return '?' + keys.map(key => `${key}=${params[key]}`).join('&');
 }
 
+/**
+ * Takes a link, looks for params already in the current URL
+ * and generates a new link with those params
+ */
+export function linkWithQueryParams(link) {
```

Comment:
I wonder if there are other places doing this same thing which we could update? Totally not blocking just figured I'd ask.

Reply:
oh good thought. Even a quick look shows me I missed one spot in our lesson overview code 🙃 I'll take a quick look to see if there are other places as well but I'll definitely update that spot

Reply:
Found a few more places in the dropdown and the editors